rdrf/rdrf/features/steps.py

```diff
@@ -214,6 +233,12 @@ def click_save_button(step):
     click(save_button)
 
 
+@step('And I save the form')
+def click_save_button(step):
+    save_button = world.browser.find_element_by_class_name("btn-primary")
```

Comment:
I couldn't see that we are using a consistent way to identify elements. The method directly above, for instance, is doing the exact same thing (clicking save button on form) using a different part of the webdriver api. Its also problematic in that you can't just use the high level natural language feature file when writing the tests - you need to know how the element is identified (class, id, xpath etc) and then what 'natural language' triggers that particular identification. Which defeats the purpose of the natural language. It would be good to talk that through and achieve a consensus on how we want to identify page elements to aid writing tests.